shell.c

```diff
@@ -0,0 +1,24 @@
+#include "shell.h"
+
+/**
+ * main - Entry point for our shell program
+ *
+ * @ac: (integer) number of argument of the program
+ * @av: (array of strings) array of parameters of the program with len=ac
+ * @envp: (array of strings) array of environment variable path.
+ * @return int 0 for success otherwise error number
+ */
+int main(int ac, char **av, char **envp)
+{
+	char line;
```

Comment:
is `line` a pointer or a character?

Reply:
good catch, it should be a pointer.

Reply:
fixed by [e9b088c](https://github.com/rakati/simple_shell/pull/1/commits/e9b088c1b8813a23ff37e30dac7754ad10274276)